combat-trainer.lic

```diff
@@ -2017,7 +2017,7 @@ class TrainerProcess
       # Dump the box so that combat-trainer lootprocess can take over
       bput("open my #{box}", 'You open', 'That is already open')
       2.times do
-        bput("dismantle my #{box}", 'You can not dismantle', 'You dump the contents', 'You move your hands')
+        bput("dismantle my #{box}", 'You can not dismantle', 'You dump the contents')
```

Comment:
❓ Why was this removed?

Reply:
My local copy did not have it. I added back in the update below.